src/pudl/metadata/fields.py

```diff
@@ -1257,31 +1561,77 @@
+        "unit": "kV",
+    },
+    "home_area_network": {
+        "type": "integer",
+        "description": (
+            "Number of AMI meters with home area network (HAN) gateway enabled."
+        ),
+    },
     "hrsg": {
         "type": "boolean",
         "description": "indicates if the boiler is a heat recovery steam generator (HRSG).",
     },
-    "inactive_accounts_included": {"type": "boolean"},
+    "inactive_accounts_included": {
+        "type": "boolean",
+        "description": (
+            "Whether the respondent includes inactive accounts in its definition of "
+            "customers used to determine SAIDI and SAIFI."
+        ),
+    },
     "income_type": {
         "type": "string",
         "description": "Type of income reported in income_statement_ferc1 table.",
     },
-    "income": {
-        "type": "number",
-        "description": "Utility income reported by income type.",
-        "unit": "USD",
-    },
     "increase_in_other_regulatory_liabilities": {
         "type": "number",
         "description": "The increase during the reporting period of other regulatory liabilities.",
         "unit": "USD",
     },
-    "incremental_energy_savings_mwh": {"type": "number", "unit": "MWh"},
-    "incremental_life_cycle_energy_savings_mwh": {"type": "number", "unit": "MWh"},
-    "incremental_life_cycle_peak_reduction_mwh": {"type": "number", "unit": "MWh"},
-    "incremental_peak_reduction_mw": {"type": "number", "unit": "MW"},
+    "incremental_energy_savings_mwh": {
+        "type": "number",
+        "description": (
+            "energy savings in the given report year resulting from new participants "
+            "in existing demand response programs and all participants in new demand "
+            "response programs."
+        ),
+        "unit": "MWh",
+    },
+    "incremental_life_cycle_energy_savings_mwh": {
+        "type": "number",
+        "description": (
+            "The estimated total changes in energy use for incremental programs and "
+            "participants over the life of the programs. DSM programs have a useful "
+            "life, and the net effects of these programs will diminish over time. "
+            "Considers the useful life of energy efficiency technology by accounting "
+            "for building demolition, equipment degradation, and program attrition."
+        ),
+        "unit": "MWh",
+    },
+    "incremental_life_cycle_peak_reduction_mwh": {
+        "type": "number",
+        "description": (
+            "The estimated total changes in peak load for incremental programs and "
+            "participants over the life of the programs. DSM programs have a useful "
+            "life, and the net effects of these programs will diminish over time. "
+            "Considers the useful life of energy efficiency technology by accounting "
+            "for building demolition, equipment degradation, and program attrition."
+        ),
+        "unit": "MWh",
```

Comment:
This field comes from the `core_eia861__yearly_energy_efficiency ` table. Peak demand reduction should be reported in MW not MWh. I think this is just a column name error and we should change it to be `incremental_life_cycle_peak_reduction_mw`